memcarduino.py

```diff
@@ -411,7 +407,7 @@ def result(passed):
 		file = arg
 		mode = "WRITE"
 	elif opt in ("-c" , "--capacity"):
-		end = arg
+		end = int(arg)
```

Comment:
This is unrelated, but current memcarduio.py is broken if you pass `-c` as `arg` is a string, and `end` needs to be a number